frontend/src/pages/MeetingRoom/MeetingRoom.tsx

```diff
@@ -101,6 +104,17 @@ const MeetingRoom = (): ReactElement => {
       />
       <RightPanel activeTab={rightPanelActiveTab} handleClose={closeRightPanel} />
       <EmojisOrigin />
+      {/* Renders the CaptionsBox directly in the meeting room on small port devices to ensure that captions
+       are always visible and accessible in the limited viewport. 
+       This placement provides an optimal user experience for accessibility on smaller screens. */}
+      {isSmallViewport && isSessionCaptioningEnabled && (
```

Comment:
Based on this, we'll render the captions box whenever the session has captioning enabled for small viewport devices. Is that consistent with larger/desktop view? Do we need to click/tap the captions button for either case?